posthog/api/test/test_decide.py

```diff
@@ -302,6 +304,26 @@ def test_session_recording_linked_flag_variant(self, *args):
         response = self._post_decide().json()
         self.assertEqual(response["sessionRecording"]["linkedFlag"], {"flag": "my-flag", "variant": "test"})
 
+    def test_session_recording_url_trigger_patterns(self, *args):
+        self._update_team(
+            {
+                "session_recording_url_trigger_config": [{"url": "/replay-examples/", "matching": "regex"}],
+                "session_recording_opt_in": True,
+            }
+        )
+
+        response = self._post_decide(origin="capacitor://localhost:8000/home").json()
+        assert response["sessionRecording"] == {
+            "endpoint": "/s/",
+            "recorderVersion": "v2",
+            "consoleLogRecordingEnabled": True,
+            "sampleRate": None,
+            "linkedFlag": None,
+            "minimumDurationMilliseconds": None,
+            "networkPayloadCapture": None,
+            "urlTriggers": [{"url": "/replay-examples/", "matching": "regex"}],
+        }
```

Comment:
Not sure why, but the test is not doing what I think it should be doing :( 

<img width="997" alt="image" src="https://github.com/user-attachments/assets/34546e16-5fe5-4ee7-8252-153ceb7cb51b">


@pauldambra  any idea why?